lib/rectangle.js

```diff
@@ -0,0 +1,25 @@
+
+var pry = require('pryjs')
+
+
+function area(rectangle){
+  return rectangle.width * rectangle.height
+};
+function perimeter(rectangle){
+  return rectangle.width * 2 + rectangle.height * 2
+};
+function diagonal(rectangle){
+  return Math.sqrt((rectangle.width * rectangle.width) + (rectangle.height * rectangle.height)).toFixed(2)
+};
+
+function square(rectangle){
+  if (rectangle.width == rectangle.height){
+  return true
+}
```

Comment:
again, watch your spacing. interviewers may judge you on your ability to follow spacing conventions (i know it seems a little silly but i've heard of it happening!)